auv_control_demos/README.md

```diff
@@ -0,0 +1,321 @@
+# AUV Control Demos
+
+This package contains examples that demonstrate how to use the controllers in
+`auv_controllers`. There are two examples. The first shows how to launch just a
+single controller on its own. The second example shows how to chain the
+controllers together to a mock hardware interface for an AUV.
+
+## Example 1: Individual Controller
+
+This example shows how to launch an individual auv controller connect them to a
+mock hardware interface. We launch the integral sliding mode controller which
+takes velocity reference and state interfaces and outputs a wrench command
+interface.
```

Comment:
I think you could probably shorten this to just something like: "This example uses the integral sliding mode controller (insert link) to demonstrate how to launch a single controller."

Reply:
Also why chose the ISMC controller for the demo? Could you launch another controller that's maybe a bit easier to use?

Reply:
> I think you could probably shorten this to just something like: "This example uses the integral sliding mode controller (insert link) to demonstrate how to launch a single controller."

done

Reply:
We'll make that change. I don't think we have strong reasoning for using the ISMC controller over a different controller for the demo.

I think there is one advantage of showing the demo with this controller as opposed to a simpler controller. If someone wants to try out using a simpler controller, they can omit parts from this demo to run that simpler controller. If we instead show the demo with a simpler controller, then it would probably be confusing trying to figure out what to include to make the demo work with the ISMC controller. That reasoning does go both ways, though.